unit2.js

```diff
@@ -0,0 +1,94 @@
+// function getOption() {
+//     var obj = document.getElementById("mySelect");
+//     document.getElementById("demo").innerHTML = 
+//     obj.options[obj.selectedIndex].text;
+//   }
+
+
+const getFilms = () => {
+    const filmUrl = "https://ghibliapi.herokuapp.com/films"
+    fetch(filmUrl)
+        .then(response => response.json())
+        .then(response => {
+            //console.log(response)
+            getFilmTitle(response);
+        })
+        .catch(error => {
+            console.log("Error! Something is not working!")
+        })
+
+}
+
+getFilms()
+
+const getFilmTitle = (response) => {
```

Comment:
This function and sendTitles function can be combined together since they both loop through array of movies. I know I always tell folks to create simple functions with one job, but this can be an exception to that rule since going through one less loop will improve time complexity of the app.